apps/aetx/src/aetx_env.erl

```diff
@@ -71,22 +72,6 @@
 %%% API
 %%%===================================================================
 
--spec contract_env(aec_blocks:height(), non_neg_integer(),
-                   non_neg_integer(), aec_keys:pubkey(),
-                   aec_pow:difficulty(), aec_blocks:block_header_hash()
-                  ) -> env().
-contract_env(Height, ConsensusVersion, Time, Beneficiary, Difficulty,
-             KeyHash) ->
-    #env{ beneficiary = Beneficiary
-        , consensus_version = ConsensusVersion
-        , context = aetx_contract
-        , difficulty = Difficulty
-        , height  = Height
-        , key_hash = KeyHash
-        , signed_tx = none
-        , time = Time
-     }.
-
 -spec tx_env_from_key_header(aec_headers:key_header(),
                              aec_blocks:block_header_hash(),
                              non_neg_integer(),
```

Comment:
Not sure why `_PrevHash` passed as argument at all, as unused. Though out of scope for this PR.

Reply:
I'm guilty of this. I had the wrong idea of the semantics of the BLOCKHASH instruction was, and it is a leftover from that implementation. No use removing it in this PR as noted.